src/framework.js

```diff
@@ -0,0 +1,28 @@
+import AppService from './app.service';
+import DashboardController from './controllers/dashboard.controller';
+
+class Framework {
+  /**
```

Comment:
**Kindly remove commented code, you can always use git history to access previous code right :slightly_smiling_face: **